source/directives/select-field.ts

```diff
@@ -298,6 +302,7 @@ module formFor {
 
           // Rather than sanitizing `$scope.model.bindable` to undefined, update the empty option's value.
           // This way users are able to choose between undefined, null, and empty string ~ see #141
+          $scope.model.bindable = formForConfiguration_.defaultSelectEmptyOptionValue;
```

Comment:
Sorry for delay... Yes I tested it really rigorously and I did not find any issues. But anyway, I'm still little bit worried that it can cause circular loop when there is some watch or something?